### DISCUSSION TITLE
Time reaction

### DISCUSSION THREAD

**Discussion by mx344** [vgalin/html2image#16] [Q&A]
I have a problem because the screenshot is taken to early, and the webpage, hasn't charged yet.
How can I delay the moment of the screenshot?

**Comment by vgalin** [comment]
Hello, please take a look at [this reply](https://github.com/vgalin/html2image/discussions/8#discussioncomment-284127) I gave on a previous discussion, it should solve the problem.

**Reply by mx344** [reply]
Yeah, thank you very much, It helped me a lot. Very useful library.
